### PR TITLE
Syncing settings and assets

### PR DESCRIPTION
**Pull request by chris-short**

### CODE REVIEW

**Comment by chris-short** [comment]
This PR is designed to take all the work done in the windows-vm branch and pushing it to master.

I'd like more eyeballs on this as the master branch is for Linux streamers.

File: obs/README.md [hunk 1]
```diff
@@ -4,28 +4,25 @@ This directory houses assets needed for streaming when using [**OBS**](https://o
...
 ## HOWTO install assets so they "just work"
 
-In an attempt to make this as simple as possible, we have built this OBS Scene to be compliant with Mac and Linux. We've tried to make the installation process just as simple. However, we now have a number of different sets of assets. The following is a template installation for the main OpenShift Streaming Scene Collection.
+In an attempt to make this as simple as possible, we have built this OBS Scene to be compliant with Mac and Linux. We've tried to make the installation process just as simple:
 
 1. Clone this repo into `/opt`
 1. In OBS remove any existing **OpenShift Streaming** Scenes from the Scene Collection menu  
 (NOTE: you won't do this the first time. Also, if you don't do this OBS will probably crash. Sadly, OBS doesn't automatically pick up Scene changes)
-1. In OBS, select Scene Collection -> Import, use the path to the OBS scene `/opt/streaming-tools/obs/openshift_streaming/basic/scenes/OpenShift_Streaming.json`, and click Add.
+1. In OBS, select Scene Collection -> Import, use the path to the OBS scene `/opt/streaming-tools/obs/OpenShift_Streaming.json`, and click Add.
```
Comment (thoraxe) [inline obs/README.md:12]: This file doesn't exist in this PR

**Review by thoraxe — CHANGES_REQUESTED** [review]
One minor change needed.

Is there not a way for all the scenes to start imported?

Also, how does this work across Linux, Mac, and Windows? Scene files have paths to assets and Windows paths don't look like Linux/Mac paths.

**Comment by chris-short** [comment]
This PR doesn't make sense at this time.

**Pull request closed without merging**